src/Microsoft.DotNet.Web.Spa.ProjectTemplates.csproj

```diff
@@ -22,7 +22,13 @@
 
   <Choose>
```

Comment:
Confirmed the dev build cases here work in local testing within the spa-templates submodule in aspnetcore as well as in an "isolated" spa-templates clone.